source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4216,6 +4216,13 @@ export class OpenUrlAction extends Action {
         this.url = Utils.getStringValue(json["url"]);
     }
 
+    render(baseCssClass: string = "ac-pushButton") {
+        super.render(baseCssClass);
+
+        // OpenUrl actions behave like a hyperlink. Make sure screenreaders treat them that way.
+        this.renderedElement.setAttribute("role", "link");
```

Comment:
Shouldn't this code be in the `Action` class' render method, and look like this:
```typescript
this.renderedElement.setAttribute("role", this.getAriaRole());
```

Reply:
I believe you're thinking about the PR from yesterday (#4059) rather than this original fix (#3914)...

Reply:
Yes, I am. Given the fix from yesterday, I think this fix should be implemented using the newly introduced getAriaRole() method. Or is this fix going into another branch that doesn't have getArialRole()?

Reply:
yeah, this is targeting `release/1.2`... that said, I think you're right. I might need to reconcile these two changes...